CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [0.16.1]
```

Comment:
```suggestion
## [0.17.0]
```
Actually, we're not making a bug fix here; we're making an API change, so we should probably label this as a minor release.